### PR TITLE
Update wmt08.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description

Structure changes in the `Results` section.

## Type of PR

- Edits _[WMT08]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

File: events/wmt08.md [hunk 1]
```diff
@@ -89,68 +89,40 @@ The **Third Workshop on Machine Translation** (**WMT08**) took place on 19 June,
...
-
-| Language pair | System | `≥ others` |
-| --- | --- | --- |
-| → Czech | `PC-TRANSLATOR` | 0.674 | 
-| → French | `RBMT4` | 0.768 | 
-| → German | `RBMT4` | 0.748 | 
-| → Spanish | `RBMT4` | 0.754 | 
-
-The results were determined with a [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement), the acceptability score. 
+The constituent score measures how often a system was judged to be better than or equal to any other system in the constituent-based evaluation.
```
Comment (cefoo) [inline events/wmt08.md:98]: I fear that only adding the info about `≥ others` and `constituent score` would leave `relative ranking` and `yes-and-no constituent judgement` out.

Perhaps these explanations are not necessary, as the definitions of these scores are already included in `wmt.md`, what do you think?

File: events/wmt08.md [hunk 2]
```diff
@@ -89,68 +89,40 @@ The **Third Workshop on Machine Translation** (**WMT08**) took place on 19 June,
 
 Full results of the shared tasks: [*Further Meta-Evaluation of Machine Translation*](https://www.statmt.org/wmt08/pdf/WMT09.pdf)
 
-### News translation
+### Translation
 
-The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` ("greater than or equal to others") score.
+The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` (“greater than or equal to others”) score, [constituent ranking](wmt.md#constituent-ranking), and [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement).
```
Comment (cefoo) [inline events/wmt08.md:94]: I would add the determiners "the" and "a".

```
[...] the `≥ others` (“greater than or equal to others”) score, the [constituent ranking](wmt.md#constituent-ranking), and a [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement).
```

File: events/wmt08.md (hunk 1 above)
Comment (liashahnazaryan) [reply events/wmt08.md:98]: I added them as the WMT article has only the explanations for how those evaluation types work and not how their scores are calculated. I suggest adding the score calculation explanations to the wmt.md as well before cutting those out of this article.

File: events/wmt08.md (hunk 2 above)
Comment (liashahnazaryan) [reply events/wmt08.md:94]: done

File: events/wmt08.md (hunk 1 above)
Comment (cefoo) [reply events/wmt08.md:98]: Yes, I agree. Let's do that.

**Pull request merged**